### PR TITLE
ci: try autopushing to PRs for kicbase

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

**Comment by sharifelgamal** [comment]
ok-to-build-image

**Comment by sharifelgamal** [comment]
ok-to-build-image

**Comment by sharifelgamal** [comment]
ok-to-build-image

File: hack/jenkins/kicbase_auto_build.sh [hunk 1]
```diff
@@ -76,43 +76,34 @@ fi
...
+	git remote add ${ghprbPullAuthorLogin} git@github.com:${ghprbPullAuthorLogin}/minikube.git
+	git fetch ${ghprbPullAuthorLogin}
+	git checkout -b ${ghprbPullAuthorLogin}-${ghprbSourceBranch} ${ghprbPullAuthorLogin}/${ghprbSourceBranch}
+
+	sed -i "s|Version = .*|Version = \"${KIC_VERSION}\"|;s|baseImageSHA = .*|baseImageSHA = \"${sha}\"|;s|gcrRepo = .*|gcrRepo = \"${GCR_REPO}\"|;s|dockerhubRepo = .*|dockerhubRepo = \"${DH_REPO}\"|" pkg/drivers/kic/types.go; make generate-docs;
+
+	git commit -am "Updating kicbase image to ${KIC_VERSION}"
+	git push ${ghprbPullAuthorLogin} HEAD:${ghprbSourceBranch}
+
+	message="Hi ${ghprbPullAuthorLoginMention}, we have updated your PR with the reference to newly built kicbase image. Pull the changes locally if you want to test with them or updateyour PR further."
```
Comment (medyagh) [inline hack/jenkins/kicbase_auto_build.sh:95]: updateyour -->  update your

**Review by medyagh — CHANGES_REQUESTED** [review]

File: hack/jenkins/kicbase_auto_build.sh (hunk 1 above)
Comment (sharifelgamal) [reply hack/jenkins/kicbase_auto_build.sh:95]: fixed

**Comment by sharifelgamal** [comment]
ok-to-build-image

**Review by medyagh — APPROVED** [review]

**Pull request merged**